Makefile.am

```diff
@@ -42,7 +42,8 @@ AM_CFLAGS = -fstack-protector -Wall -pedantic \
         -Wall -W -D_FORTIFY_SOURCE=2 \
         -Wno-missing-field-initializers \
         -std=c11 \
-        -DSYSCONFDIR=\"$(sysconfdir)\"
+        -DSYSCONFDIR=\"$(sysconfdir)\" \
+	    -D_POSIX_C_SOURCE=201112L
```

Comment:
Yes. Needs fixing.

Reply:
@ahkok, so i made `Makefile.am` use spaces and tabs consistently throughout and git decided it had to be re-written. It's slightly more pleasant [to view](https://github.com/busykai/clr-boot-manager/blob/kai/shim-systemd/Makefile.am) now.